src/content/1.7/basics/keeping_up-to-date/migration.md

```diff
@@ -0,0 +1,678 @@
+several times if your shop is multi-language – once per language.
+Forcing the IDs provided by the CSV file will help you link your
+localized data with the original one.
+
+##### Import main configuration
+
+To import data, start by uploading your CSV file. It will be done
+automatically when you select your file from the filesystem.
+
+Before CSV selection or upload:
+
+{{< figure src="/images/1.7/upgrade-migration/image65.png" >}}
+
+Some warnings may be displayed depending on the type of data to import.
+They can provide useful advice about forbidden or allowed rules.
+
+Once a CSV is selected:
+
+{{< figure src="/images/1.7/upgrade-migration/image81.png" >}}
+
+Double check the field & multiple-value options for your CSV structure.
+Default values should work in most of case, but incorrect values will
+make the import fail.
+
+Checking the “Delete all &lt;data&gt; before import” (see screenshot
+above) option could be useful to clean the demonstration content from
+your shop before importing your data. But if you export several CSV of
+the same type of data, make sure to disable it before proceeding with
+the second and following files.
+
+##### Column assignment
+
+The second step allows you to link the old columns with the new ones.
+
+{{< figure src="/images/1.7/upgrade-migration/image77.png" >}}
+
+In case a column from the CSV should not be taken in account, you can
+ask PrestaShop to bypass it by choosing the first option of the list:
+“Ignore this column”.
+
+When you’re ready to proceed, click on “Import”.
+
+##### Import progress
+
+From that moment, you can just wait for the process to complete. The
+time it will take depends on the number of lines to import, and the
+chosen options.
+
+{{< figure src="/images/1.7/upgrade-migration/image32.png" >}}
+
+To avoid reaching the maximum execution time of a request, the process
+will divide your data in several chunks which will be processed in
+separate HTTP requests.
+
+The import process will start by checking the time needed to process one
+chunk. Depending on the result, more or less objects will be handled
+during the next HTTP requests. This gives a nice balance between
+performance and stability.
+
+### Images
+
+Depending on how you migrated your data, (i.e without the import feature
+of PrestaShop), your images may be missing from the new shop. If they
+are not present, you may want to retrieve them from the old.
+
+Images are stored on the filesystem and their link to a product is
+stored in the database. Restoring them depends on how you migrated your
+data:
+
+-   You kept the same IDs when you imported the products
+
+If the IDs are exactly the same between the two shops, copying the
+folder \`/img/p\` will be enough to get the images.
+
+Also, you must transfer the data of the tables &lt;PREFIX&gt;**images** /
+&lt;PREFIX&gt;**images_lang** / &lt;PREFIX&gt;**images_shop** in which the
+link between your images and products IDs is stored.
```

Comment:
If you want these paragraphs aligned with the bullet point, you need to apply indentation